CODEOWNERS

```diff
@@ -1,4 +1,4 @@
 # SPDX-License-Identifier: Apache-2.0
 
-# Firefly committers
-* @hyperledger-labs/firefly-committers
+# firefly-cli maintainers
```

Comment:
does this need to be updated to `firefly-samples maintainers`?

Reply:
Good catch. This was copy/pasta. Fortunately it's just a label but I'll update it anyway.

Reply:
🙏🏻